tasks.py

```diff
@@ -37,18 +37,29 @@ def build(ctx):
 
 @task
 def release_build(ctx):
-    ctx.run('pip install "$(grep build setup.cfg | cut -f2 -d-)"')
+    ctx.run('python -m pip install "$(grep build setup.cfg | cut -f2 -d-)"')
```

Comment:
`python -m pip install .[development]` would be a cleaner way to install the `build` package (the `build` module is listed as a dependency in the `development` group in `setup.cfg`. Maybe even put `build >= 0.7` in a separate `build` category in `setup.cfg` (instead of `development`) and use `python -m pip install .[build]` to install the build-time modules.